client/controllers/incidentReports/annotatedContent.coffee

```diff
@@ -64,6 +62,22 @@ Template.annotatedContent.events
         # Set reactive variable that's handed down from curatorSourceDetails and
         # shared with the incidentTable templates to the clicked annotation's ID
         instance.data.selectedAnnotationId.set(annotationId)
+        data =
+          incidentId: annotationId
+          source: instance.data.source
+          scrolled: instance.scrolled
+          showPopup: instance.showPopup
+          popupDelay: POPUP_DELAY
```

Comment:
I think it would make more sense to move POPUP_DELAY to the popup template than pass it as a parameter.